packages/ui/src/modal-side/ModalSide.css

```diff
@@ -0,0 +1,141 @@
+.ui-ModalSide--AriaDialog {
+  outline: none;
+}
+
+.ui-ModalSide--Overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background: var(--theme-black-opacity-40);
+}
+
+.ui-ModalSide--Modal {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  width: 60%;
+  outline: none;
+  border-left: var(--layout-border-width-small) solid
+    var(--surface-color-border-default);
+  box-shadow: -8px 0 20px var(--surface-opacity-10);
+  background: var(--theme-gray-white);
+  display: flex;
+  flex-direction: column;
+  padding: var(--layout-padding-large);
+
+  &[data-entering] {
+    animation: modal-slide-in 300ms reverse ease-in;
+  }
+
+  &[data-exiting] {
+    animation: modal-slide-out 300ms reverse ease-in;
```

Comment:
Curious to understand how this animation performs on slower machines? Do we think a 

will-change: auto is needed?

Reply:
`Warning: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.`
https://developer.mozilla.org/en-US/docs/Web/CSS/will-change